bip-0085.mediawiki

```diff
@@ -356,6 +380,39 @@ GPG capable smart-cards SHOULD be loaded as follows: The encryption slot SHOULD
 
 However, depending on available slots on the smart-card, and preferred policy, the CERTIFY capable key MAY be flagged with CERTIFY and SIGNATURE capabilities and loaded into the SIGNATURE capable slot (for example where the smart-card has only three slots and the CERTIFY capability is required on the same card). In this case, the SIGNATURE capable sub-key would be disregarded because the CERTIFY capable key serves a dual purpose.
 
+===DICE===
+
+Application number: 89101'
+
+The derivation path format is: <code>m/83696968'/89101'/{sides}'/{rolls}'/{index}'</code>
+
+    2 <= sides <= 2^32 - 1
+    1 <= rolls <= 2^32 - 1
+
+Use this application to generate PIN numbers or any other numeric secret.
```

Comment:
I wonder if it's worth adding a note on the fact that users could use a 36-sided die to generate, say, an alphanumeric password as Base64 and Base85 have more characters than are supported by conservatives systems like legacy databases?

Reply:
Happy to review if you attempt it.

Reply:
Added.